policy/README.md

```diff
@@ -238,17 +241,23 @@ An individual `Rule` object is defined by the following fields:
 | `count` | Fleet counts based on regions. Rule `max`/`min` refers to number of devices.                                  |
 | `time`  | Individual limitations on time spent in one or more vehicle-states. Rule `max`/`min` refers to increments of time in [Rule Units](#rule-units). |
 | `speed` | Global or local speed limits. Rule `max`/`min` refers to speed in [Rule Units](#rule-units).                  |
+| `rate`  | Fees or subsidies based on regions and time spent in one or more vehicle-states. Rule `rate_amount` refers to the rate in [Rule Units](#rule-units).      |
 | `user`  | Information for users, e.g. about helmet laws. Generally can't be enforced via events and telemetry.          |
 
 ### Rule Units
+Note that all rates are in the `currency` defined in the [Policy](#policy)
 
 | Name      | Description         |
 | --------- | ------------------- |
-| `seconds` | Seconds             |
-| `minutes` | Minutes             |
-| `hours`   | Hours               |
-| `mph`     | Miles per hour      |
-| `kph`     | Kilometers per hour |
+| `seconds`             | Seconds             |
+| `minutes`             | Minutes             |
+| `hours`               | Hours               |
+| `mph`                 | Miles per hour      |
+| `kph`                 | Kilometers per hour |
+| `rate_per_event`      |  Rate applied once when the vehicle enters a matching status.   |        
+| `rate_per_minute`     |  Rate applied for each minute while the vehicle is in the matching status.   |
+| `rate_per_hour`     |  Rate applied for each hour while the vehicle is in the matching status.   |       
+| `rate_per_day`        |  Rate applied once per day if the vehicle is in the matching status at any point.   |        
```

Comment:
If there is a deployment policy that is `rate_per_day` which is $0.25 in geo1 but only $0.5 in geo2 and a vehicle is deployed to both over the course of a day what happens? Do you imagine this would be in an ordered policy and the first matching one applies? That seems reasonable and would let one take the max deployment fee. 

I think this could be made clearer with an example. I believe you are planning on adding examples to the `examples.md` file?

Reply:
Oh, I see your examples notion page now. 👍

Reply:
This is probably your plan but adding those examples to `examples.md` when submitting your PR to omf seems like it would help people understand this proposal.

Reply:
Yep! That's the plan once we have aligned on which examples to include.